src/integrationTest/resources/application.yaml

```diff
@@ -41,7 +41,7 @@ ftp:
 scheduling.enabled: false
 
 tasks:
-  upload-letters: ${UPLOAD_LETTERS_INTERVAL:1}
+  upload-letters-interval-ms: ${UPLOAD_LETTERS_INTERVAL:1000} #
```

Comment:
Do we need `#` at the end?

Reply:
Also we could update `UPLOAD_LETTERS_INTERVAL` to `UPLOAD_LETTERS_INTERVAL_IN_MS` ?

Reply:
Would be nicer 👍